content/en/continuous_integration/tests/_index.md

```diff
@@ -60,10 +60,12 @@ There's also information about the wall time of the most recent test suite run,
 
 Hovering over the commit author avatar shows detailed information about the latest commit.
 
-#### Benchmark test regressions
-A benchmark test run is marked as a regression when its duration is five times the standard deviation above the mean for the same test in the default branch. The mean of the default branch is calculated over the 200 most recent test runs. A benchmark test has @test.type:benchmark.
+#### Test regressions
+A test run is marked as a regression when its duration is five times the mean and greater than the max duration for the same test in the default branch. The mean and the max of the default branch is calculated over the last week of test runs.
```

Comment:
can we say, "...when its duration is both five times the mean and greater than the max..."